src/appleseed-max-impl/appleseedinteractive/interactivetilecallback.h

```diff
@@ -56,7 +56,12 @@ class InteractiveTileCallback
         IIRenderMgr*                    iimanager,
```

Comment:
Unrelated to this PR but I couldn't help from noticing: `iimanager` doesn't make much sense as a name. I guess the first I is for "Interface" and the second is for "Interactive"? In that case maybe `irender_manager`? Or `render_manager`?

Reply:
True, The naming comes from the class interface declared in IIRenderManager.h, part of the Max SDK.
`This class represents the abstract (interface) for an interactive rendering
manager.`